exercises/simple-linked-list/example.js

```diff
@@ -0,0 +1,72 @@
+function ListElement (value, next) {
```

Comment:
So, this is a nitpick, but the canonical README says:

> Write a simple linked list implementation that uses Elements and a List.

I think with that in mind, the classes should be changed to be called `Element` and `List`.  Since these aren't reserved words in JavaScript, we should be OK.